unleash-yggdrasil/src/state.rs

```diff
@@ -32,3 +35,49 @@ pub enum SdkError {
     StrategyEvaluationError,
     StrategyParseError,
 }
+
+#[cfg(test)]
+mod test {
+    use crate::strategy_parsing::compile_rule;
+
+    use super::*;
+    use unleash_types::client_features::Context;
+
+    #[test]
+    fn converting_a_context_to_enriched_context_assumes_now_for_time_if_not_set() {
+        let context = Context::default();
+        let enriched_context = EnrichedContext::from(context, "test".into());
```

Comment:
`EnrichedContext::from` is what Yggdrasil uses to convert a user facing context into its own internal extended representation